CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# How to contribute
```

Comment:
This could be in its own  separate PR?

Reply:
I think it seems correct to include the documentation about the pre-commit hook in the same commit that adds it.
Either way is OK. It might be that if logically 2 commits can be simply made into 1 commit, and it's just as clear, it's a bit better.

Reply:
I moved it to a separate commit. So it's one PR with two commits now. :slightly_smiling_face: